src/bun.js/bindings/napi.cpp

```diff
@@ -970,6 +970,9 @@ static JSC::ErrorInstance* createErrorWithCode(JSC::VM& vm, JSC::JSGlobalObject*
 
     // we don't call JSC::createError() as it asserts the message is not an empty string ""
     auto* error = JSC::ErrorInstance::create(vm, globalObject->errorStructure(type), message, JSValue(), nullptr, RuntimeType::TypeNothing, type);
+    if (!error) {
+        return nullptr;
+    }
```

Comment:
```suggestion
```
JSC assumes the error is non-null (they dereference it immediately)

Reply:
i see, this calls the overload that doesnt throw